HW0/P5/P5.txt

```diff
@@ -1 +1,25 @@
-# Your answers here.
```

Comment:
I don't think the answer is Log base 2. It is the case for powers of 2 but for a general time T, you need to account for an extra second. I believe the answer is int(log_2(N)) where int rounds up the number to the next biggest integer.